matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/network/ApiPath.kt

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright (c) 2021 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.matrix.android.sdk.api.network
+
+import org.matrix.android.sdk.internal.network.NetworkConstants
+
+enum class ApiPath(val path: String, val method: String) {
```

Comment:
I think it's error prone as we have to add path in different places each time we add a new API.
Maybe we could generate the paths from the Retrofit interfaces.

Reply:
I agree, we could improve that in the future